src/lexer.rs

```diff
@@ -87,7 +193,7 @@ impl Lexer {
         let kind = self.read_next_kind();
         let end = self.offset();
 
-        while kind == Kind::WhiteSpace {
+        if kind == Kind::WhiteSpace {
```

Comment:
I have failed to write recursive code :D

Reply:
xD